test/test.js

```diff
@@ -68,14 +68,6 @@ describe('hotkey', function () {
       assert.notInclude(elementsActivated, 'button3')
     })
 
-    it("doesn't trigger when user is focused on a input or textfield", function () {
-      setHTML(`
-      <button id="button1" data-hotkey="b">Button 1</button>
-      <input id="textfield" />`)
-      document.getElementById('textfield').dispatchEvent(new KeyboardEvent('keydown', {key: 'b'}))
-      assert.deepEqual(elementsActivated, [])
-    })
-
```

Comment:
I feel like we want to keep this test, I think we can instead add a special case saying that if the `data-hotkey-scope` is a form field we count it, otherwise we ignore keypresses from it.

In other words form fields can be ignored unless they are explicitly the `data-hotkey-scope`